# HG changeset patch
# User Sean Farley <sean@farley.io>
# Date 1452040850 28800
#      Tue Jan 05 16:40:50 2016 -0800
# Node ID f03080aa3226c638f6e8429c802688521882a9f4
# Parent  c921ba940436a9c0c1260b8c85083a9595e1ebbc
use @@VAR@@ for substitution later

diff --git a/SRC/make.PETSC b/SRC/make.PETSC
--- a/SRC/make.PETSC
+++ b/SRC/make.PETSC
@@ -1,27 +1,28 @@
-AR = ar
+AR = /usr/bin/ar
 ARFLAGS = cr
-#If your system does not have ranlib, then replace next statement with 
-#RANLIB = echo
-RANLIB = ranlib
+RANLIB = /usr/bin/ranlib
 
-CC = /sandbox/hzhang/externalpackages/mpich2-1.0.3/linux-gnu-intel-hong/bin/mpicc
+CC = @@CC@@
 CLD = $(CC)
 # Set f77 to echo if you do not have a FORTRAN compiler
-F77 = /sandbox/hzhang/externalpackages/mpich2-1.0.3/linux-gnu-intel-hong/bin/mpif77
+F77 = @@FC@@
 
 F77LD = $(F77)
 FFXN = -DAdd_
 FSUFFIX = F
 
-MPIF77 = /sandbox/hzhang/externalpackages/mpich2-1.0.3/linux-gnu-intel-hong/bin/mpif77
-MPICC = /sandbox/hzhang/externalpackages/mpich2-1.0.3/linux-gnu-intel-hong/bin/mpicc
+MPIF77 = @@MPIFC@@
+MPICC = @@MPICC@@
 
 # Please include mpi header file path, if needed
-CFLAGS = -O3 -DLittleEndian $(PMLCGDEF) $(MPIDEF) -D$(PLAT) 
+CFLAGS = @@CFLAGS@@
 
-CLDFLAGS =  -O3 
+CLDFLAGS =
 
-FFLAGS = -O3 $(PMLCGDEF) $(MPIDEF) -D$(PLAT) -I.
-F77LDFLAGS =  -O3 
+FFLAGS = @@FFLAGS@@
+F77LDFLAGS =
 
-CPP = cpp -P
+CPP = @@CPP@@
+
+SHAREDFLAGS = @@SHAREDFLAGS@@
+PREFIX = @@PREFIX@@
